scripts/push-schema-changes.js

```diff
@@ -76,6 +76,7 @@ async function main() {
       { repo: "volt-v2" },
```

Comment:
@damassi it looks like this is already here?

Reply:
He already reverted 😬

Reply:
perfect!